crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -527,6 +527,22 @@ impl<'w, 's> Commands<'w, 's> {
         self.queue.push(RunSystem::new(id));
     }
 
+    /// Runs the system by referring it. This method will register the system newly called with,
```

Comment:
These docs definitely need to be improved. Could you list out the full limitations of this approach, compared to the register and then run workflow? Then I'll wordsmith them for grammar and clarity :)

Reply:
I will try make it.